libraries/WebServer/examples/WebServer/README.md

```diff
@@ -28,9 +34,9 @@ It features
 
 Currently, this example supports the following targets.
 
-| Supported Targets | ESP32 | ESP32-S2 | ESP32-C3 |
-| ----------------- | ----- | -------- | -------- |
-|                   | yes   | yes      | yes      |
+| Supported Targets | ESP32 | ESP32-S2 | ESP32-S3 | ESP32-C3 |
+| ----------------- | ----- | -------- | -------- | -------- |
+|                   | yes   | no       | yes      | yes      |
```

Comment:
What is the limitation on ESP32-S2?

Reply:
It compiles and maybe works, but I since I initially wrote the example I could not test it on S2.

Reply:
I have tested to be sure today, work fine on all chips.